### PR TITLE
Add tagging to postsubmit jobs

### PR DESCRIPTION
**Pull request by iamsubratp**

### CODE REVIEW

File: container_images/daisy-builder/main.sh [hunk 1]
```diff
@@ -17,13 +17,15 @@ PROJECT="$1"
...
 GCS_OUTPUT_BUCKET="$4"  # Destination for artifacts
+VERSION=""
 
 function generate_new_version() {
   local VERSION_OUT
   if ! VERSION_OUT=`/versiongenerator --token-file-path=${GITHUB_ACCESS_TOKEN} --org=${REPO_OWNER} --repo=${REPO_NAME} 2>&1`; then
       echo "could not generate version: ${VERSION_OUT}"
       return 1
   fi
+  VERSION=${VERSION_OUT}
```
Comment (hopkiw) [inline container_images/daisy-builder/main.sh]: global is set here...

File: container_images/daisy-builder/main.sh [hunk 2]
```diff
@@ -81,6 +83,24 @@ function generate_build_workflow() {
   echo -e "$config" > "$WF"
 }
 
+tag_commit() {
+  LATEST_VERSION=$(generate_new_version)
+  if [[ "$LATEST_VERSION" != "$VERSION" ]]; then
```
Comment (hopkiw) [inline container_images/daisy-builder/main.sh]: ...also returned here by the function, and then compared. won't these always be equal?

File: container_images/daisy-builder/main.sh [hunk 3]
```diff
@@ -81,6 +83,24 @@ function generate_build_workflow() {
...
+    echo "not valid build $LATEST_VERSION != $VERSION"
+    return 1
+  fi
+  TAGGER_CMD="/tagger --token-file-path=${GITHUB_ACCESS_TOKEN} --tag=${LATEST_VERSION} --sha=${PULL_BASE_SHA} "
+  TAGGER_CMD+="--org=${REPO_OWNER} --repo=${REPO_NAME}"
```
Comment (hopkiw) [inline container_images/daisy-builder/main.sh]: suggest escape the newline, rather than +=

File: container_images/daisy-builder/main.sh (hunk 2 above)
Comment (iamsubratp) [reply container_images/daisy-builder/main.sh]: fixed

File: container_images/daisy-builder/main.sh (hunk 3 above)
Comment (iamsubratp) [reply container_images/daisy-builder/main.sh]: done

**Pull request merged**